R/methods-XCMSnExp.R

```diff
@@ -1129,14 +1129,14 @@ setMethod("[[", "XCMSnExp",
 #'     parent class.
 #'
 #' @author Johannes Rainer
-setMethod("bin", "XCMSnExp", function(object, binSize = 1L, msLevel.) {
-    if (hasAdjustedRtime(object) | hasFeatures(object) |
-        hasChromPeaks(object)) {
+setMethod("bin", "XCMSnExp", function(x, binSize = 1L, msLevel.) {
```

Comment:
was there a need to rename object to x ?

Reply:
yes, because `bin` is now imported from `ProtGenerics` - there were multiple `bin` definition and now this was unified into one - with `x` instead of `object`.